src/main/scala/dpla/ingestion3/mappers/providers/InMapping.scala

```diff
@@ -0,0 +1,140 @@
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "created")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(stringOnlyTimeSpan)
+
+  override def description(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "description")
+
+  override def extent(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "extent")
+
+  override def format(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "medium")
+      .flatMap(_.splitAtDelimiter(";"))
+
+  // TODO: This is in the crosswalk but not in ingest1. It is also identical to format mapping.  Keep?
+  override def genre(data: Document[NodeSeq]): ZeroToMany[SkosConcept] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "medium")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyConcept)
+
+  override def identifier(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "identifier")
+
+  override def language(data: Document[NodeSeq]): ZeroToMany[SkosConcept] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "language")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyConcept)
+
+  override def publisher(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "publisher")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyAgent)
+
+  override def preview(data: Document[NodeSeq]): ZeroToMany[EdmWebResource] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "source")
+      .map(stringOnlyWebResource)
+
+  override def place(data: Document[NodeSeq]): ZeroToMany[DplaPlace] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "spatial")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyPlace)
+
+  override def provider(data: Document[NodeSeq]): ExactlyOne[EdmAgent] = agent
+
+  override def rights(data: Document[NodeSeq]): AtLeastOne[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "accessRights")
+
+  override def subject(data: Document[NodeSeq]): ZeroToMany[SkosConcept] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "subject")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyConcept)
+
+  override def temporal(data: Document[NodeSeq]): ZeroToMany[EdmTimeSpan] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "temporal")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(stringOnlyTimeSpan)
+
+  override def title(data: Document[NodeSeq]): AtLeastOne[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "title")
+
+  // TODO: Watch out for this field during QA.
+  //  Ingest1 has the following enrichments that may indicate the need for some logic here:
+  //  "/enrich-type?default=image" ?
+  //  "/enrich-type?send_rejects_to_format=true"
+  override def `type`(data: Document[NodeSeq]): ZeroToMany[String] =
```

Comment:
good catch. The solution might be to map the type field to format and then apply a format allowFilter on the type values. 

Not sure if we want to apply a default type value though...that seems awkward.